modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/FilePageStoreManager.java

```diff
@@ -1264,16 +1265,13 @@ public int pageSize() {
     }
 
     /** Dumps related partition files and index file to diagnostic dir. */
-    public void dumpPartitionFiles(int grpId, long... pageIds) {
+    public void dumpPartitionFiles(File baseDumpDir, int grpId, long... pageIds) {
         try {
-            String path = "db/dump/" + igniteCfg.getConsistentId() + "/" + grpId;
-            File dumpDir = U.resolveWorkDirectory(igniteCfg.getWorkDirectory(), path, false);
+            File dumpDir = new File(baseDumpDir, Integer.toString(grpId));
```

Comment:
It would be much less convenient, I'd rather document it

Reply:
Imagine that the dump contains 100 numeric directories for 100 groups intermingled with few other directories (like 'wal'). It would be pretty difficult to find the needed non-numeric directories in such a listing

Reply:
It only contains a single cache group for the majority of cases